### DISCUSSION TITLE
Very beginner

### DISCUSSION THREAD

**Discussion by Terrapi95** [probonopd/MiniDexed#639] [Q&A] [answered]
Hello everyone, I'm going to sound like a very beginner but that's what I am with the raspberry pi. 
I have been playing the synthesizer for 40 years and I was very attracted by the minidexed having had a dx5 a very long time ago. Unfortunately I am unable to make it work (I probably missed a step).
I took a raspberry pi 4 model B, a 1602A lcd, a rotary encoder, a pcm5100 dac. I have an 8gb micro SD card formatted in fat32 on which I entered all the decompressed files from the zip folder MiniDexed_2024-03-25-082445b.zip. After powering up the red LED is ok, the green one flashes then turns off, the LCD backlight is ok, the DAC power supply too. nothing is displayed on the LCD and the encoder has no effect. Listening after connecting a mini lab MK2 midi keyboard I have a unique typical FM sound which responds well to different notes.
If you could help me I would be grateful.
In advance thank you

Richard

**Comment by Banana71** [comment]
Hello Richard,
nice that you're here.
If you can hear something and the miniDexed reacts to notes, there can't be that much wrong.
Have you adapted the minidexed.ini?
Does your display have a backpack (I2C) or is it connected directly to the RPi?
Can you please upload your minidexed.ini here. (rename minidexed.ini to minidexed.txt and then upload)

```
# HD44780 LCD
LCDEnabled=1
LCDPinEnable=17
LCDPinRegisterSelect=4
LCDPinReadWrite=0
LCDPinData4=22
LCDPinData5=23
LCDPinData6=24
LCDPinData7=25
LCDI2CAddress=0x00

# SSD1306 LCD
# For a 128x32 display, set LCDColumns=20; LCDRows=2
# For a 128x64 display, set LCDColumns=20; LCDRows=4
SSD1306LCDI2CAddress=0x3c
SSD1306LCDWidth=128
SSD1306LCDHeight=32
SSD1306LCDRotate=0
SSD1306LCDMirror=0

# Default is 16x2 display (e.g. HD44780)
LCDColumns=16
LCDRows=2
```

**Comment by Terrapi95** [comment]
I'm sorry but there is no link behind the "here"

**Comment by Terrapi95** [comment]
Oh sorry for the question, the LCD is directly connected to the RPi.

**Reply by Banana71** [reply, marked as answer]
- Resistor:
Have you soldered the resistor for the contrast adjustment to the display?

_"If you are not using An an i2c "backpack" board, you can use a 10k potentiometer between the contrast pin (V0) and GND. The contrast varies with a changing environment temperature. If you don't have a 10k potentiometer at hand, a 2.2k resistor may also do the job, but you may have to experiment with the exact value. "_

- minidexed.ini:
does your minidexed.ini look like this?

```
# HD44780 LCD
LCDEnabled=1
LCDPinEnable=17
LCDPinRegisterSelect=4
LCDPinReadWrite=0
LCDPinData4=22
LCDPinData5=23
LCDPinData6=24
LCDPinData7=25
LCDI2CAddress=0x00

# SSD1306 LCD
# For a 128x32 display, set LCDColumns=20; LCDRows=2
# For a 128x64 display, set LCDColumns=20; LCDRows=4
SSD1306LCDI2CAddress=0x00
SSD1306LCDWidth=128
SSD1306LCDHeight=32
SSD1306LCDRotate=0
SSD1306LCDMirror=0

# Default is 16x2 display (e.g. HD44780)
LCDColumns=16
LCDRows=2
```

**Comment by Terrapi95** [comment]
Hello, thank you for your answer. Indeed I made an error on the value of the resistance! So I took the opportunity to set up two potentiometers, one for the display and the other for the retro lighting.
Thank you very much for your help and I will not hesitate to return to you in case of needs. I can also send you photos of the finished project.

Musically
Richard

**Reply by probonopd** [reply]
Great that it is working for you now @Terrapi95. Would be great to see your finished project!
Thanks @Banana71 for the hint with the resistor.

**Comment by Terrapi95** [comment]
Hello everyone, as promised the mini Dexed in photos. for the moment it works very well with a master keyboard, but is not recognized by my computer or my MAO software.
![IMG_20240420_150830](https://github.com/probonopd/MiniDexed/assets/166457738/e2b80a7f-e0e7-44bd-ab8b-94098332bc82)
![IMG_20240420_150850](https://github.com/probonopd/MiniDexed/assets/166457738/621555ea-b157-483a-ac40-590decdf5f35)
![IMG_20240420_150915](https://github.com/probonopd/MiniDexed/assets/166457738/729b7085-b4ee-45d4-a350-c1b889238c19)
![IMG_20240420_180316](https://github.com/probonopd/MiniDexed/assets/166457738/61f20f49-5e30-403b-800e-82296524bc24)

**Reply by probonopd** [reply]
> is not recognized by my computer or my MAO software

Depending on your model of RPi, you may need an additional USB-MIDI to serial-MIDI adapter which you can build from a Raspberry Pi Pico as described in the Wiki.
